chainerx_cc/chainerx/float16_test.cc

```diff
@@ -235,5 +235,26 @@ TEST(NativeFloat16Test, Float16DivideI) {
     }
 }
 
+TEST(NativeFloat16Test, FloatComparison) {
+    for (Float16 x : GetFloat16Values()) {
+        for (Float16 y : GetFloat16Values()) {
+#define run_compare_check(op)                                                \
```

Comment:
Could you follow the style in this test (also regarding the naming conventions)? https://github.com/chainer/chainer/blob/master/chainerx_cc/chainerx/routines/type_util_test.cc